src/plugins/themes.js

```diff
@@ -62,11 +62,11 @@ function getModuleInfo(packageName) {
 		log.warn(`Specified theme ${colors.yellow(packageName)} is not installed in packages directory`);
 		return;
 	}
-	if (!module.lounge) {
+	if (!module.thelounge) {
 		log.warn(`Specified theme ${colors.yellow(packageName)} doesn't have required information.`);
 		return;
 	}
-	return module.lounge;
+	return module.thelounge;
```

Comment:
Unrelated to this PR, and also I realize that it doesn't hurt to be on the safe side, but now that the `install` command takes care of checking the metadata, maybe we should assume that if a package reaches this point, it has the correct metadata, no?

Reply:
It's better to be on the safe side.